src/engine-actions/wait-for-swap-commitment.js

```diff
@@ -8,7 +8,7 @@ class CanceledSwapError extends Error {}
  * Waits for a swap to enter the ACCEPTED state and errors if it won't happen
  *
  * @param {Bytes} swapHash - hash for a swap
- * @returns {Invoice} - invoice for swap
+ * @returns {Date} - creation date of the HTLC
```

Comment:
@treygriffith i'm thinking we would want to return a string instead `.toISOString()`?

Reply:
I think a date makes sense here - it's a native JS type, so passing it shouldn't be a problem.